build/config.mak

```diff
@@ -7,6 +7,8 @@ EPSILON_VERSION ?= 13.1.0
 OMEGA_VERSION ?= 1.20.0
 # USERNAME ?= N/A
 EPSILON_APPS ?= calculation rpn graph code statistics probability solver atom sequence regression settings external omega
+EPSILON_APPS_CAN_BE_HIDE ?= 1 1 1 1 1 1 1 1 1 1 0 0 1
+
```

Comment:
This seems like a bad way to do config, as it depends on the order of the apps in `EPSILON_APPS`. Maybe store the name of the vars which can be hidden ?